.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    types: [ opened, reopened, synchronize, labeled ]
+
+env:
+  HELM_VERSION: v3.15.2
+  PGO_VERSION: 5.7.4
```

Comment:
Can these be GitHub Actions vars instead of hard coded in the workflow? That way we can change them without having to make code changes.